.github/workflows/linux.yml

```diff
@@ -1,31 +1,24 @@
 name: Test
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+    - "docs/**"
+  pull_request:
+    paths-ignore:
+    - "docs/**"
 
 jobs:
   test:
 
-    runs-on: ${{ matrix.os }}-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu, windows, macos]
+        os: [ubuntu-latest]
```

Comment:
Why keeping os in the matrix when there is just one?

Reply:
This could definitely be inlined, I just kept to avoid updating the caching and coverage keys

Reply:
Done